tasmota/settings.h

```diff
@@ -738,8 +738,10 @@ struct {
   uint8_t       shd_leading_edge;          // F5B
   uint16_t      shd_warmup_brightness;     // F5C
   uint8_t       shd_warmup_time;           // F5E
+  uint8_t       rulesAfterFlash;           // F5F - flag for running rules only once after firmare flash
+
```

Comment:
If you want only execution once then there is no need for this. If a user does initial installation of formware the settings are written completely. The next restarts will not execute settings init again so the commands will not execute either.

Reply:
without the **rulesAfterFlash** at each reboot the setup function is processed and this will involve execution of 
**SettingsSave(0)**, so  it will write the flash at each reboot, or not ?

The execute command for applying rules is a fast process why should it trigger the watchdog ?

What is exactly the new CMND feature field in the template ?